landing_page/views.py

```diff
@@ -23,11 +24,31 @@ def about(request):
 def contact(request):
     return render(request, "landing_page/contact.html")
 
-  
+
 def profile(request):
-    return render(request, "accounts/profile.html")
+    if request.user.is_authenticated:
```

Comment:
let's avoiding nesting this much - do "if request.user.is_authenticated is False', return them to login

Reply:
or, way better like you did above
if not request.user.is_authenticated: